tools/deploy.sh

```diff
@@ -110,7 +110,8 @@ mkdir -p "${TEMP_BMO_OVERLAY}"
 mkdir -p "${TEMP_IRONIC_OVERLAY}"
 
 KUSTOMIZE="${SCRIPTDIR}/tools/bin/kustomize"
```

Comment:
maybe can be done in a follow up but if you invert the lines between KUSTOMIZE and KUSTOMIZE_BUILD you can reuse the latter to build the former